### PR TITLE
Feature/remove private

### PR DESCRIPTION
**Pull request by eproctor93**

### CODE REVIEW

**Comment by eproctor93** [comment]
Changed the private wording back to what it was at Brians request.

**Comment by Pheonnexx** [comment]
LGTM

File: service/templates/layout.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
     <div class="phase-banner">
       <p>
         <strong class="phase-tag">BETA</strong>
-        <span>This is a new service - your <a href="mailto:digital-register-feedback@digital.landregistry.gov.uk">feedback</a> will help us to improve it.</span>
+        <span>This service is in private beta. Please give us your <a rel="external" href="mailto:digital-register-feedback@digital.landregistry.gov.uk">Feedback</a>.</span>
```
Comment (andymantell) [inline service/templates/layout.html:44]: @AndyPorter as well as adding back in the word "private", the wording here has been changed to an earlier version. Is this correct? Proto 18 went with words along the lines of "your feedback will help us to improve it"

Comment (AndyPorter) [inline service/templates/layout.html:44]: I think it should still be - This is a new service - your feedback will help us improve it.
I'll try to find out where this change/decision has come from

Comment (andymantell) [inline service/templates/layout.html:44]: @AndyPorter Adding back in the word private is necessary to satisfy the 90 thing isnt it? Seeing as we aren't in public beta? The question I was raising was that the rest of the wording was reverted too

Comment (AndyPorter) [inline service/templates/layout.html:44]: ah, are you suggesting - 'this service is in private beta [your feedback will help us improve it]?  [] = link

Comment (andymantell) [inline service/templates/layout.html:44]: @AndyPorter Yes!

**Comment by AndyPorter** [comment]
do not merge this change.
gov uk pattern for private and public beta is to have the phrase - 'this service is in private beta [your feedback will help us improve it]? [] = link
so do not change it. :)

**Pull request closed without merging**